automatic/paint.net/tools/ChocolateyInstall.ps1

```diff
@@ -1,23 +1,28 @@
 ﻿$ErrorActionPreference = 'Stop'
 
-$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$toolsPath = Split-Path -parent $MyInvocation.MyCommand.Definition
+$silentArgs     = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
+$fileType   = 'msi'
 
 $packageArgs = @{
-  PackageName    = $env:ChocolateyPackageName
-  fileType       = 'exe'
-  file64         = Get-Item $toolsPath\*.exe
-  silentArgs     = '/auto'
-  validExitCodes = @(0, 1641, 3010)
```

Comment:
The valid exit codes should not be removed. Please add these back and make sure they are used.

Reply:
Why did you resolve this comment? You have not made the change I requested.